README-minishift.md

```diff
@@ -188,17 +188,31 @@ oc create clusterrolebinding federation-admin \
     --serviceaccount="federation-system:default"
 ~~~
 
-Deploy the [cluster registry](https://github.com/kubernetes/cluster-registry) and the namespace where clusters are registered
-(`kube-multicluster-public`):
+Clone the Federation V2 repo and checkout the tag 0.0.4:
```

Comment:
Ditto about clone/checkout: if you add the relevant commit of the federation-v2 submodule, the `git clone --recurse-submodules` should have already taken care of this - maybe just add a note explaining what happened during the clone.